docs/content/en/docs/tasks/cluster/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -136,67 +136,85 @@ and then you will run the [upgrade cluster command]({{< relref "baremetal-upgrad
-Forcing reconcile Git repo with latest commit
-GitOps not configured, force reconcile flux git repo skipped
-Resuming Flux kustomization
-GitOps field not specified, resume flux kustomization skipped
-```
-
-During the upgrade process, EKS Anywhere pauses the cluster controller reconciliation by adding the paused annotation `anywhere.eks.amazonaws.com/paused: true` to the EKS Anywhere cluster, provider datacenterconfig and machineconfig resources, before the components upgrade. After upgrade completes, the annotations are removed so that the cluster controller resumes reconciling the cluster.
-
-Though not recommended, you can manually pause the EKS Anywhere cluster controller reconciliation to perform extended maintenance work or interact with Cluster API objects directly. To do it, you can add the paused annotation to the cluster resource:
-
-```bash
-kubectl annotate clusters.anywhere.eks.amazonaws.com ${CLUSTER_NAME} -n ${CLUSTER_NAMESPACE} anywhere.eks.amazonaws.com/paused=true
-```
-
-After finishing the task, make sure you resume the cluster reconciliation by removing the paused annotation, so that EKS Anywhere cluster controller can continue working as expected.
-
-```bash
-kubectl annotate clusters.anywhere.eks.amazonaws.com ${CLUSTER_NAME} -n ${CLUSTER_NAMESPACE} anywhere.eks.amazonaws.com/paused-
-```
+* kubectl CLI: The cluster lifecycle feature lets you use kubectl to talks to the Kubernetes API to create a workload cluster. To use kubectl, run:
+   ```bash
+   kubectl apply -f eksa-w01-cluster.yaml --kubeconfig mgmt/mgmt-eks-a-cluster.kubeconfig
+   ```
+  As noted earlier, adding the `--kubeconfig` option tells `eksctl` to use the management cluster identified by that kubeconfig file to create a different workload cluster.
+
+* **GitOps**: See [Manage separate workload clusters with GitOps]({{< relref "../../tasks/cluster/cluster-flux.md#manage-separate-workload-clusters-using-gitops" >}})
+
+* **Terraform**: See [Manage separate workload clusters with Terraform]({{< relref "../../tasks/cluster/cluster-terraform.md#manage-separate-workload-clusters-using-terraform" >}})
+
+  >**NOTE**:For kubectl, GitOps and Terraform:
+  >
+  > You can't add additional hardware when updating workload cluster.
+  >
+  >  If you want to update multiple workload clusters, you should update them one by one, and make sure its completely up and running successfully by checking tinkerbell machines, before attempting to update the next workload cluster. Attempting to update multiple workload clusters without letting the requests complete successfully can cause hardware race conditions. In a situation where sufficient hardware is not provisioned, the creation would fail at the CAPT level.
+  > 
+  > This also applied to sending upgrade requests to the same workload cluster.
+
+* eksctl CLI: To create a workload cluster with eksctl, run:
+
+  ##### With hardware CSV
+  
+  ```
+  eksctl anywhere upgrade cluster -f cluster.yaml --hardware-csv <hardware.csv>
```

Comment:
```suggestion
    eksctl anywhere upgrade cluster \
         # --hardware-csv <hardware.csv>  \  # uncomment to add more hardware
         -f cluster.yaml 
```
We could do that same thing here. Drop "With/Without Hardware CSV..." and use one example.